<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fieldOptionsDialog</class>
 <widget class="QDialog" name="fieldOptionsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Новая игра</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/pics/icon.png</normaloff>:/pics/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберите размер поля:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonEasy">
     <property name="text">
      <string>Новичок. Размер поля 9х9, 10 бомб</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonMedium">
     <property name="text">
      <string>Любитель. Размер поля 16х16, 40 бомб</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonHard">
     <property name="text">
      <string>Профессионал. Размер поля 30х16, 99 бомб</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonManual">
     <property name="text">
      <string>Свои настройки</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ширина (9-30):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBoxWidth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>9</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Высота (9-24):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxHeight">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>9</number>
         </property>
         <property name="maximum">
          <number>24</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Число бомб (1-700):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxBombs">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>700</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
